src/org/mirah/builtins/services/org.mirah.builtins.ExtensionsProvider

```diff
@@ -0,0 +1 @@
+org.mirah.builtins.Builtins
```

Comment:
Could you put a comment in this file about it? The SPI Spec supports `#` prefixed line comments.